frontend/e2e/validation.spec.ts

```diff
@@ -0,0 +1,60 @@
+import { test, expect } from "@playwright/test"
+
+test.describe("validation errors", () => {
+  test("Wrong character range", async ({ page }) => {
+    await page.goto(
+      "http://localhost:5173/amending-laws/eli/bund/bgbl-1/2017/s419/2017-03-15/1/deu/regelungstext-1/articles/hauptteil-1_art-1/edit/hauptteil-1_art-1_abs-1_untergl-1_listenelem-2_inhalt-1_text-1_%C3%A4ndbefehl-1",
```

Comment:
Todo: you can use a relative URL so we don't hardcode host and port. Same for the other URLs